be/src/olap/storage_engine.cpp

```diff
@@ -776,18 +783,18 @@ void StorageEngine::_clean_unused_txns() {
     }
 }
 
-OLAPStatus StorageEngine::_do_sweep(const string& scan_root, const time_t& local_now,
+OLAPStatus StorageEngine::_do_sweep(const FilePathDesc& scan_root_desc, const time_t& local_now,
                                     const int32_t expire) {
```

Comment:
Cold data will have no trash or garbage, not deal with it.

Reply:
It may be deleted later, but I think is safer when migration operation is online for the first time.